gordon/schema/validate.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gordon import exceptions
+
+import json
+import os
+
+import jsonschema
+
+DEFINITIONS_DIRECTORY = os.path.join(os.path.dirname(__file__), 'definitions')
+
+
+class JsonValidator(object):
```

Comment:
yeah I'd agree, I'd prefer the `class JsonValidator:` style

Reply:
I have no preference, especially since I'm not really using any of object's methods, but the style guide does say the opposite https://google.github.io/styleguide/pyguide.html?showone=Classes#Classes